src/components/Images/__snapshots__/Images.spec.js.snap

```diff
@@ -0,0 +1,71 @@
+    media="(min-width: 600px)"
+    type="image/jp2"
+  />
+  <source
+    data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,fl_progressive:semi,h_100,q_auto:eco,w_100/v1/something.com/otherthing.jp2 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,fl_progressive:semi,h_100,q_auto:eco,w_100/v1/something.com/otherthing.jp2 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,fl_progressive:semi,h_100,q_auto:eco,w_100/v1/something.com/otherthing.jp2 3x"
+    media="(max-width: 599px)"
+    type="image/jp2"
+  />
+  <source
+    data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,fl_progressive:semi,h_400,q_auto:eco,w_400/v1/something.com/otherthing.jpeg 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,fl_progressive:semi,h_400,q_auto:eco,w_400/v1/something.com/otherthing.jpeg 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,fl_progressive:semi,h_400,q_auto:eco,w_400/v1/something.com/otherthing.jpeg 3x"
+    media="(min-width: 1200px)"
+    type="image/jpeg"
+  />
+  <source
+    data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,fl_progressive:semi,h_300,q_auto:eco,w_300/v1/something.com/otherthing.jpeg 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,fl_progressive:semi,h_300,q_auto:eco,w_300/v1/something.com/otherthing.jpeg 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,fl_progressive:semi,h_300,q_auto:eco,w_300/v1/something.com/otherthing.jpeg 3x"
+    media="(min-width: 900px)"
+    type="image/jpeg"
+  />
+  <source
+    data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,fl_progressive:semi,h_200,q_auto:eco,w_200/v1/something.com/otherthing.jpeg 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,fl_progressive:semi,h_200,q_auto:eco,w_200/v1/something.com/otherthing.jpeg 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,fl_progressive:semi,h_200,q_auto:eco,w_200/v1/something.com/otherthing.jpeg 3x"
+    media="(min-width: 600px)"
+    type="image/jpeg"
+  />
+  <source
+    data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,fl_progressive:semi,h_100,q_auto:eco,w_100/v1/something.com/otherthing.jpeg 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,fl_progressive:semi,h_100,q_auto:eco,w_100/v1/something.com/otherthing.jpeg 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,fl_progressive:semi,h_100,q_auto:eco,w_100/v1/something.com/otherthing.jpeg 3x"
+    media="(max-width: 599px)"
+    type="image/jpeg"
+  />
+  <img
+    alt="alt text"
+    className="Image lazyload testImage"
+    data-src="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,fl_progressive:semi,q_auto:eco/v1/something.com/otherthing.jpeg"
+  />
+</picture>
+`;
```

Comment:
really sorry to pile on here, but i have a different concern with this pr. right off the bat, it seems very strange to me that the optimal way to send images would be 12 source tags, each with a data-src attribute over 400 characters long, and an img tag. 

this single image is over 6400 characters, or about 6.4kb, of just html. the front page has 22 <img> tags, each of which would be replaced with 6.4kb of html. 22 * 6.4kb = 140.8kb. our current front page is 189kb (before gzip), so this pr would almost double the size of the html being transferred. given that one of our OKR's is first contentful paint i would really like to see some timing on this.

if you look at the [cloudinary docs](https://cloudinary.com/documentation/image_transformations#automatic_format_selection) they have "format=auto" and "quality=auto" settings which offload most of the work that this file is doing. they also pepper in odd little 'tips' like this one:

> “When automatic format (f_auto) is used together with automatic quality (q_auto), the PNG format may be selected when the automatic quality algorithm decides that it better fits the specific image. This allows delivering better looking and economical image files.”

so first of all, this goes against what i thought i knew about image formats -- i was unaware that png's would ever be faster than jpeg's, let alone these weird new formats like webp/jpeg-2000. and right now our generated html never makes a png, so either they are wrong or there are times when we're fetching something less than optimal. 

this goes back to an early thought I had about this pr, which is that it feels like we're duplicating the work that cloudinary is supposed to be doing for us of choosing the best format and quality settings based on device size, os, etc. without performance testing we have no way of knowing if we are right or not; and maybe this is laziness but I sort of assume that cloudinary will generally do a better job than we will, because this is one of the main selling points for their business.

**tl;dr** before we merge this I think we should do some quick tests to see if this is indeed faster than letting cloudinary determine format and quality on a handful of browsers. the important things to measure would be first contentful paint (due to our okr) and the final above-the-fold page load time.

Reply:
Agree^

Reply:
@kenning @zach-ethos Okay I changed up this PR a little bit. It should now use the automatic format setting that cloudinary offers and through testing it looks like it works pretty well, this was a good call. I had to leave in the DPR setting as the auto setting in their lib was not working as well manual. 
I also used this opportunity to add in LQIP to the images which should help with first loads and content shifting around. I added a blur-up to make the transition a little smoother between the images.
This is the gist of what we build now:
* 4 source tags, one for each media query.
* 1 img tag that has the LQIP as `srcSet` and also a fallback image for old browsers in the `src`

```html
<picture>
  <source media="(min-width: 1200px)" data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,f_auto,fl_progressive:semi,h_624,q_auto:eco,w_636/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,f_auto,fl_progressive:semi,h_624,q_auto:eco,w_636/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,f_auto,fl_progressive:semi,h_624,q_auto:eco,w_636/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 3x"/>
  <source media="(min-width: 900px)" data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,f_auto,fl_progressive:semi,h_428,q_auto:eco,w_428/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,f_auto,fl_progressive:semi,h_428,q_auto:eco,w_428/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,f_auto,fl_progressive:semi,h_428,q_auto:eco,w_428/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 3x"/>
  <source media="(min-width: 600px)" data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,f_auto,fl_progressive:semi,h_332,q_auto:eco,w_332/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,f_auto,fl_progressive:semi,h_332,q_auto:eco,w_332/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,f_auto,fl_progressive:semi,h_332,q_auto:eco,w_332/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 3x"/>
  <source media="(max-width: 599px)" data-srcset="https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_1.0,f_auto,fl_progressive:semi,h_250,q_auto:eco,w_375/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 1x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_2.0,f_auto,fl_progressive:semi,h_250,q_auto:eco,w_375/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 2x, https://res.cloudinary.com/getethos/image/upload/c_fill,dpr_3.0,f_auto,fl_progressive:semi,h_250,q_auto:eco,w_375/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg 3x"/>
  <img class="Images-module--Image--1EVuH  lazyload " src="https://res.cloudinary.com/getethos/image/upload/c_fill,f_auto,fl_progressive:semi,q_auto:eco,t_unsupported/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg" srcSet="https://res.cloudinary.com/getethos/image/upload/c_fill,f_auto,fl_progressive:semi,h_250,q_auto:eco,t_lqip,w_375/v1560273411/01_Lifestyle/life-insurance-01-family-playing-in-bed.jpg" alt="Family playing in bed"/>
</picture>
```

As you can see this is ~2900 characters per image, vs the ~9k from before with 12 source tags. The impact is very low from my testing on FCP. A page with many images like `/blog` comes out to about 50k more characters which is zipped to ~700 bytes. 

I went pretty soft on LQIP so they are usually between 5 -10 kb each. But we could take a more aggressive approach using either a simple default placeholder, or using greyscale versions of the images . (~1.5 - 3 kb).

Reply:
we medium now